components/constellation/lib.rs

```diff
@@ -14,15 +14,16 @@ extern crate serde;
 mod browsingcontext;
 mod constellation;
 mod event_loop;
+pub mod logger;
```

Comment:
Is it possible to remove `pub` here in order to only export FromCompositorLogger and FromScriptLogger?

Reply:
Seems like not. Attempting that results in `private module` error on `logging` in `servo/lib`.

Reply:
I see. Thank you for reviewing  :)